plugins/oauth/hooks.js

```diff
@@ -62,6 +64,21 @@ module.exports = Object.assign({}, StoreMixin, {
     }]);
   },
 
+  dcosInstallCommand(instructions) {
+    let steps = instructions.split('&&');
+    steps.splice(-1, 0, ' \n dcos auth login ');
```

Comment:
This seems highly error prone to me and I suggest to refactor this using the `mount` package.